ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -125,7 +125,8 @@ public boolean accept(File dir, String name) {
     */
     this(datadir, scheduler, log, config, factory, recovery, hardDelete, metrics,
         new Journal(datadir, 2 * config.storeIndexMaxNumberOfInmemElements,
-            config.storeMaxNumberOfEntriesToReturnFromJournal), time, incarnationId);
+            config.storeMaxNumberOfEntriesToReturnFromJournal), time, sessionId, incarnationId,
+        CLEAN_SHUTDOWN_FILENAME);
```

Comment:
I'm going to be using the index implementation for the target index (the one that is the result of compaction) - that will require a different clean shutdown file name that does not clash with the one for the main index.